### PR TITLE
Fix missing conf for tickets

### PR DESCRIPTION
**Pull request by john-botella**
Add missing conf for tickets

TICKET_REMOVE_TRACK_URL : Remove track url in ticket email
this option remove the link used view the progress of the ticket in the specific interface in email sended.
TODO : Allow use dolibarr mail template system instead of hard coded mail template and remove this option and add a select template model option

TICKET_REMOVE_THIRDPARTY_MAIL_COPY : Do not send a copy of the e-mail to the address of the default third party.
For many companies, the primary contact email is not suitable for ticket-related correspondence. Sending a copy may have the effect of spamming an unintended service.

rebuild a part of admin options for tickets

Before
image

After
image

### CODE REVIEW

**Comment by atm-arnaud** [comment]
@atm-john thank you for the TICKET_REMOVE_THIRDPARTY_MAIL_COPY conf. I never understand this copy by default 👍

File: htdocs/admin/ticket.php [hunk 1]
```diff
@@ -48,6 +49,55 @@
...
+$item = $formSetup->newItem('TICKET_REMOVE_TRACK_URL');
+$item->setAsYesNo();
+$item->nameText = $langs->transnoentities('TicketsRemoveTrackUrl');
+$item->helpText = $langs->transnoentities('TicketsRemoveTrackUrlHelp');
+
+$item = $formSetup->newItem('TICKET_REMOVE_THIRDPARTY_MAIL_COPY');
+$item->setAsYesNo();
+$item->nameText = $langs->transnoentities('TicketsRemoveThirdPartyEmailCopy');
+$item->helpText = $langs->transnoentities('TicketsRemoveThirdPartyEmailCopyHelp');
+
```
Comment (eldy) [inline htdocs/admin/ticket.php:100]: What about TICKET_LIMIT_VIEW_ASSIGNED_ONLY and TICKET_NOTIFY_AT_CLOSING ?

Comment (atm-john) [reply htdocs/admin/ticket.php:100]: merge mistake I guess...

Comment (atm-john) [reply htdocs/admin/ticket.php:100]: TICKET_NOTIFY_AT_CLOSING is already  present and TICKET_LIMIT_VIEW_ASSIGNED_ONLY there is only save action but form seem to not exist.

**Comment by eldy** [comment]
PR too old and not mergeable. Thanks to resubmit it (1 option = 1 pr) if a similar feature has not been already merged in recent version.

**Pull request closed without merging**